src/index.tsx

```diff
@@ -6,16 +6,16 @@ import App from './App'
 import * as serviceWorker from './serviceWorker'
 import { store } from './features/store'
 import { BrowserRouter as Router } from 'react-router-dom'
-import { theme } from './features/theme'
-import { ThemeProvider } from '@material-ui/core/styles'
 import { MuiPickersUtilsProvider } from '@material-ui/pickers'
+import { ThemeProvider } from '@material-ui/core/styles'
+import { basicTheme } from './features/theme'
 import LuxonUtils from '@date-io/luxon'
 import './features/locale'
 
 ReactDOM.render(
   <React.StrictMode>
     <Provider store={store}>
-      <ThemeProvider theme={theme}>
+      <ThemeProvider theme={basicTheme}>
```

Comment:
why basic theme? you have two themes: light and dark. Both inherits from basic, that's true, but basic theme is an abstract theme and cannot be used

Reply:
If is use basic theme like abstract theme I have this behavior after several theme change
![image](https://user-images.githubusercontent.com/22051751/95859403-3eb87280-0d67-11eb-8ba2-7f1e3e85f720.png)
You can reproduse this behavior if checkout to 759d commit. So now I use light and dark theme like nested theme

Reply:
anyway I don't understand what you mean. I see that you use light and dark as derived themes, I'm happy with it. I'm not happy with the fact that you use pass basic theme into theme provider. My expectation is that you need to pass either light or dark theme but not the basic one. Please explain why?

Reply:
Becouse I didn't know how switching theme without this bug. Now, I find a way how to fix this